src/app/query/bitcoin/btc/btc-crypto-asset.hooks.ts

```diff
@@ -0,0 +1,49 @@
+import type { BtcCryptoAssetInfo } from '@leather-wallet/models';
+
+import { BTC_DECIMALS } from '@shared/constants';
+import { createMoney } from '@shared/models/money.model';
+
+import { cryptoAssetFactory } from '@app/query/common/crypto-asset.model';
+import { useCryptoCurrencyMarketDataMeanAverage } from '@app/query/common/market-data/market-data.hooks';
+
+import type { BtcCryptoAsset } from '../../common/crypto-asset.model';
+import { useBtcCryptoAssetBalanceNativeSegwit } from '../balance/btc-balance-native-segwit.hooks';
+
+const btcCryptoAssetInfo: BtcCryptoAssetInfo = {
+  decimals: BTC_DECIMALS,
+  hasMemo: true,
```

Comment:
what does `hasMemo` mean? 🤔

Reply:
Memo is nec or optional with certain blockchains.

> What is a memo? A memo is an identifier code required in addition to a receive address when sending certain cryptocurrencies. Each memo code is tied to a unique wallet and helps to ensure that the transaction is sent to the correct recipient.

Reply:
ah, I just thought there is no memo in bitcoin

Reply:
Sorry, yeah, good point. I'm not sure why it has been set to true for Bitcoin. Unless someone chimes in here with a reason, I will change it to false. We don't show a memo field rn for Bitcoin, you are right.